crypto/evp/evp_lib.c

```diff
@@ -881,6 +881,7 @@ int EVP_MD_meth_set_input_blocksize(EVP_MD *md, int blocksize)
     md->block_size = blocksize;
     return 1;
 }
```

Comment:
I would leave this file alone,  these format changes are targeting dead code that is likely to be completely removed in the next major release.

Reply:
It's OK though!